ambari-server/src/main/java/org/apache/ambari/server/state/ServiceComponentImpl.java

```diff
@@ -558,14 +558,15 @@ public void deleteServiceComponentHosts(String hostname, DeleteHostComponentStat
       ServiceComponentHost sch = getServiceComponentHost(hostname);
       LOG.info("Deleting servicecomponenthost for cluster" + ", clusterName=" + getClusterName()
           + ", serviceName=" + getServiceName() + ", componentName=" + getName()
-          + ", recoveryEnabled=" + isRecoveryEnabled() + ", hostname=" + sch.getHostName());
+          + ", recoveryEnabled=" + isRecoveryEnabled() + ", hostname=" + sch.getHostName() + ", cuurentState=" + sch.getState());
       if (!sch.canBeRemoved()) {
-        throw new AmbariException("Could not delete hostcomponent from cluster"
+        throw new AmbariException("Could not delete hostcomponent from cluster, component in undeletable state"
             + ", clusterName=" + getClusterName()
             + ", serviceName=" + getServiceName()
             + ", componentName=" + getName()
             + ", recoveryEnabled=" + isRecoveryEnabled()
-            + ", hostname=" + sch.getHostName());
+            + ", hostname=" + sch.getHostName()
+            + ", currentState=" + sch.getState());
```

Comment:
While the components have several states like desired and current, the current status is something what anyone will see by default and is used as generic `state`

`currentState` => `state`

Reply:
Done. Thanks,